script/gyb/gyb

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python2.7
+import gyb
+gyb.main()
```

Comment:
I still don't like including `gyb`. It's no more work to copy-and-paste additions than it is to add them to the gyb files—and it's simpler and requires less knowledge.